house/house.js

```diff
@@ -27,19 +48,37 @@ BillFetcher.prototype.parseData = function (data, cb) {
     }
     var a = []
     for (var i = 0; i <  result.floorschedule.category.length; i++) {
-      a.push(result.floorschedule.category[i]['floor-items'])
+      for (var j = 0; j < result.floorschedule.category[i]['floor-items'].length; j++) {
+        for (var k = 0; k < result.floorschedule.category[i]['floor-items'][j]['floor-item'].length; k++) {
+          var obj = {}
+          obj['bill-number'] = result.floorschedule.category[i]['floor-items'][j]['floor-item'][k]['legis-num'][0]
+          Object.assign(obj, result.floorschedule.category[i]['floor-items'][j]['floor-item'][k]['$'])
+          obj['bill-link'] = result.floorschedule.category[i]['floor-items'][j]['floor-item'][k]['files'][0]['file'][0]['$']['doc-url']
+          obj['description'] = result.floorschedule.category[i]['floor-items'][j]['floor-item'][k]['floor-text'][0]
```

Comment:
nit: use more JS-like camel case in objects so you don't have to do the kinda awkward [ ]